modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-jvm[]
+//------------------------------------------------------//
+
+//========================//
+//API DETAILS: ALL METRICS//
+//========================//
+//
+// API DETAILS: Header-level metrics
+//-------------------------
+//tag::api-detail-header[]
+
+//TODO: Api -> API (SATHISH)
+Type::
+API implementation type (*API*)
+
+//TODO: verify for Insights; info is from API Manager docs (SATHISH)
+Status::
+Status code that indicates whether the governance policies and SLA tiers are functional (*Active*) or not
++
+For all codes, see information on xref:api-manager::latest-overview-concept.adoc#what-api-manager-looks-like[tracking registration status] in the API Manager UI documentation.
+
+//TODO: Url -> URL (SATHISH)
+Implementation URL:: 
+URL of deployed API integration
+
+//TODO: Runtime version -> Runtime Version (SATHISH)
+//TODO: issue: value can be empty; SEE BOOKS API IN STGX (SATHISH)
+//(SATHISH)
+Runtime Version::
+Mule Gateway or Flex Gateway runtime version used by the API implementation 
+
+//TODO: issue: value can be empty; SEE BOOKS API IN STGX 
+API Instance ID::
+Instance identifier for an API implementation incorporated into a Flex Gateway or API Gateway runtime
+
+//TODO: version -> Version (SATHISH)
+API Version::
+Version of the API implementation
++
+For more information, see xref:api-manager::api-instance-landing-page.adoc[].
+//end::api-detail-header[]
+//-----------------------//
+//
+//-------------------------
+// API DETAILS: Overview tab
+//-------------------------
+//tag::api-detail-overview[]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO: issue: 2 charts named Policy Violations
+//TODO: Policy Violations -> Policy Violation Count (or similar) (SATHISH)
+Policy Violations::
+Number of requests blocked because of API policy violations
+
+Request Latency::
+Latency, in milliseconds, for requests to this API at specific time points
++
+The chart provides values for the average latency and the p90, p95, p99 percentiles. 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Volume::
+Number of requests to this API at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Failed Requests::
+Number of unsuccessful requests to this API at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: issue: 2 charts named Policy Violations (SATHISH)
+Policy Violations::
+Number of requests blocked at specific time points because of API policy violations
++
+For information about policies, see xref:api-manager::manage-policies-overview.adoc[].
```

Comment:
```suggestion
//TODO: issue: 2 charts named Policy Violations (SATHISH)
Policy Violations::
Number of requests blocked at specific time points because of API policy violations
+
For information about policies, see xref:api-manager::manage-policies-overview.adoc[].
```
@sathishpaulleo